src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -40,7 +40,7 @@ case class WorkspaceRequest (
 case class Workspace (
                       namespace: String,
                       name: String,
-                      bucketName: String,
+                      workspaceId: String,
```

Comment:
I am torn here, I think we are going to need to expose the bucket name for the workspace. I originally thought we could hide the bucket but users might want to upload files.

Would it work to have a val in the Workspace object which constructs the bucket name from the workspaceId?